src/main/java/org/zalando/nakadi/service/converter/VersionOneConverter.java

```diff
@@ -59,13 +59,13 @@ public NakadiCursor convert(final String eventTypeStr, final Cursor cursor)
         } catch (final NumberFormatException ex) {
             throw new InvalidCursorException(CursorError.INVALID_OFFSET);
         }
-        final List<Timeline> timelines = eventTypeCache.getTimelinesOrdered(eventTypeStr);
-        final EventType eventType = eventTypeCache.getEventType(eventTypeStr);
+        final List<Timeline> timelines = timelineService.getActiveTimelinesOrdered(eventTypeStr);
```

Comment:
When converting cursors one should be able to convert any timeline, not just active ones